WordPress/Jetpack/Classes/ViewRelated/WordPress Migration/Welcome/MigrationWelcomeBlogTableViewCell.swift

```diff
@@ -0,0 +1,103 @@
+import UIKit
+import SwiftUI
```

Comment:
it doesn't look like we need this import here

Reply:
Good catch! I wanted to implement that cell using SwiftUI then I changed my mind. Because `Image` view doesn't support fetching images from the network and `AsyncImage` is iOS 15+.

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/19533/commits/3c76d627afe437fa16b46fc40bad4d81252bf110